contributing.md

```diff
@@ -61,40 +72,57 @@
    git push origin new-branch
 ```
 
-### 6. Send a pull request on GitHub
+#### 6. Send a pull request on GitHub
  * Go to the `new-branch` of your forked `project`
  * Click `Compare & pull request`
  * Leave a comment
    * If you are solving an issue (e.g. `#17`), add `Closes #17` in the comment, the issue will automatically be closed when the pull request is merged.
 
-### 7. Delete branch locally and/or remotely after pull request is merged on GitHub
+#### 7. Delete branch locally and/or remotely after pull request is merged on GitHub
  * Deleting your local branch from the command line: `git branch -d new-branch`
  * Additionally if you want to delete your remote branch: `git push origin : new-branch`
 
-### Reference
-* [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
-* [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+## PR conventions
+When submitting a PR, please add one of the following prefixes depending on the topic you are addressing:
+
+    [ENH]: Enhancement, new functionality
+    [BUG]: Bug fix
+    [DOC]: Additions/updates to documentation
+    [TST]: Additions/updates to tests
+    [BLD]: Updates to the build process/scripts
+    [PERF]: Performance improvement
+    [CLN]: Code cleanup
 
+In addition:
+- Please reference the relevant GitHub issues in your commit message using GH1234 or #1234.
+- Include a subject line with < 80 chars.
+- Optionally, include a commit message body, leaving one blank line with the subject line.
 
 ## PR review rules
  * Input from two reviewers is needed to merge a pull request. One reviewer approves the merge and the other reviewer merges the pull request.
 
-
 ## Issue guidelines
+
 ### New issue
 Discovering an issue is great, here's what you need to do when you discover an issue:
 * Search if the issue has already been created.
 * If yes and open refer to existing issue.
 * If yes and closed reopen issue with descriptive comment.
 * If no, create the issue by:
    * Following our issue [template.](https://github.com/pandas-dev/pandas/blob/master/.github/ISSUE_TEMPLATE.md)
+
 ### Existing issue
 * Read comments.
 * Find out if anyone is working on it, if no, offer to do it. If yes, see if you can be of help.
+
 ### Reporting bugs
 * Give information about the version and the operating system you are running.
 * Show the steps to reproduce bug.
 * Add log.
 
+## References
+* [GitHub forking](https://gist.github.com/Chaser324/ce0505fbed06b947d962)  
+* [Git cherry-pick](https://git-scm.com/docs/git-cherry-pick)
+* [Pandas contributing guides](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html)
 
 *To read more tips on how to open issues, PRs and do code reviews, read the things we have been learning throughout the mentoring program [here](https://github.com/python-sprints/pandas-mentoring/blob/master/LEARNING_POINTS.md).
```

Comment:
Extra `*` at the beginning of the line. Looks like it missed it previous PR review, can you take care of this as well. 
Thanks.

Reply:
Which `*` are you refering to? The one before "To read..."?

Reply:
Yup

Reply:
This was there on purpose to make it look like a footnote, but I changed it to another item in the list of references since it was not clear. How do you think it looks now?